src/amchart/CommonXY.js

```diff
@@ -214,6 +214,7 @@
             theme: "none",
             type: "xy",
             automargins: false,
+            path: "//www.amcharts.com/lib/3/",
```

Comment:
actually, the problem doesn't seem to be occurring in dermatology.html (i thought i had checked that) but is in playground.html and theme editor 

http://rawgit.com/mlzummo/Visualization/PLAYGROUND_RGBC/demos/playground.html

so perhaps were doing something wrong here... the error gives us an undefined in in the img url if you you load up AmChart Scatter or AmChart Bubble